src/sdk/model/__tests__/vdc.test.int.ts

```diff
@@ -103,6 +103,27 @@ test('Can get vDC and verify required properties', async() => {
   });
 });
 
+test('Can get vDC summary', async() => {
+  return Vdc.getVdc(inventoryVdc.uuid).then( async function(vdc) {
+    return vdc.getSummary().then(function(vdcSummary) {
+      expect(vdcSummary).toBeDefined();
+      expect(vdcSummary.allocationCpu).toBeDefined();
+      expect(vdcSummary.allocationMemory).toBeDefined();
+      expect(vdcSummary.configuredCpu).toBeDefined();
+      expect(vdcSummary.configuredMemory).toBeDefined();
+      expect(vdcSummary.configuredDisk).toBeDefined();
+      expect(vdcSummary.numberOfVapps).toBeDefined();
+      expect(vdcSummary.numberOfVms).toBeDefined();
+      expect(vdcSummary.reservedCpu).toBeDefined();
+      expect(vdcSummary.reservedMem).toBeDefined();
+      expect(vdcSummary.consumedCpu).toBeDefined();
+      expect(vdcSummary.consumedMem).toBeDefined();
+      expect(vdcSummary.consumedDisk).toBeDefined();
+      expect(vdcSummary.provisionedDisk).toBeDefined();
```

Comment:
@ariquinones can you also add some assertions on the toString and json methods so that we have full test coverage